include/links-default.php

```diff
@@ -84,10 +84,17 @@ public function add_paged_to_link( $url, $page ) {
 	public function get_language_from_url( $url = '' ) {
 		if ( empty( $url ) ) {
 			$url = pll_get_requested_url();
+
+			if ( empty( $url ) ) {
```

Comment:
You use a early return for performance? Otherwise it wouldn't pass the `preg_match` anyway, right?

Reply:
Yes, it prevents calling `get_languages_list()` + `preg_match()` + `implode()`.